napari/_qt/_tests/test_qt_window.py

```diff
@@ -103,3 +107,42 @@ def test_menubar_shortcut(make_napari_viewer):
     v.window._toggle_menubar_visible()
     assert not v.window.main_menu.isVisible()
     assert v.window._main_menu_shortcut.isEnabled()
+
+
+@skip_on_win_ci
```

Comment:
Why is this being skipped on Windows?

Reply:
All of our screenshot tests are skipped on windows afaik (don't know the history of why, but they *do* fail on windows ci). In fact, this fixture is misnomed, since it's specifically about screenshot tests on windows ci:

```py 
skip_on_win_ci = pytest.mark.skipif(
    sys.platform.startswith('win') and os.getenv('CI', '0') != '0',
    reason='Screenshot tests are not supported on windows CI.',
)
```

Reply:
The test I used as base for this (https://github.com/napari/napari/blob/0ffc9e827ac0566365f0ec9d440c49e532ae5644/napari/_qt/_tests/test_qt_viewer.py#L225-L229) also has the `skip_on_win_ci`. Since the test were failing I put it also in the new test. Locally the test passes for me on my Windows machine:

![imagen](https://user-images.githubusercontent.com/16781833/236258310-95bb7744-1ed5-4ab9-b3a0-4a4f949a8063.png)

As mentioned, maybe the `skip_on_win_ci` `reason` should be updated to a more generic message and a new alias mark definition should be created with the current reason related to the screenshot tests? Maybe the new alias could be something like `skip_screenshots_on_win_ci`?

Reply:
Ah, I see. Thank you both for explaining, I agree it's not the best name. I remove my objection 😄